nasa/earthdata-varinfo
# DAS-2165 - Add a config file parameter to generate_collection_umm_var method

## Description

- An optional config file parameter added to VarInfoFromNetCDF4
- Unit tests added to verify that the argument is passed in 
- Updates to Workflow 1 in how_to_publish_to_cmr_with_earthdata_varinfo.ipynb to include the config_file optional argument

## Jira Issue ID
- DAS-2165

## Local Test Steps
- Use the README.md to setup the local conda environment
- Run make develop
- Run make test
- All tests should pass

## PR Acceptance Checklist
